frontend/package.json

```diff
@@ -4,16 +4,19 @@
   "private": true,
   "dependencies": {
     "@date-io/date-fns": "^2.6.2",
-    "@material-ui/core": "^4.9.11",
+    "@material-ui/core": "^4.12.3",
+    "@material-ui/data-grid": "^4.0.0-alpha.37",
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.54",
     "@material-ui/pickers": "^3.2.10",
+    "@material-ui/x-grid-data-generator": "^4.0.0-alpha.37",
```

Comment:
You have added some new dependencies but I dont see you using them anytwhere. Are you sure it's necessary? Or it s just making the application heavier and harder to updrade with no benefit? 

Please remove the unnecessary things

Reply:
I have been threading softly with trying to remove dependencies, I will be removing it

Reply:
Still planning to emove this:    ` "@material-ui/x-grid-data-generator": "^4.0.0-alpha.37",`

Try to keep new dependencies to a minimum. a lot of stuff just adds bloat